repo/content/index/packindex_test.go

```diff
@@ -166,23 +168,37 @@ func testPackIndex(t *testing.T, version int) {
 		t.Fatalf("unable to build: %v", err)
 	}
 
+	if err := b4.BuildStable(&buf4, version); err != nil {
```

Comment:
nit: `require.NoError(t, b4.BuildStable())` is usually sufficient in such cases. It will print the line number on failure.